app/helpers/application_helper.rb

```diff
@@ -8,13 +8,11 @@ def html_options
     { class: 'htmlOptions' }
   end
 
-  # rubocop:disable RescueStandardError
   def i18n_set?(key)
     I18n.t key, raise: true
-  rescue
+  rescue StandardError
```

Comment:
What is the error we should be rescuing from here?

Reply:
Seems like there could be multiple different options here: https://guides.rubyonrails.org/i18n.html#using-different-exception-handlers, but since we're passing a key here, I think it'd be safe to assume we want to handle the `MissingTranslationData` exception.